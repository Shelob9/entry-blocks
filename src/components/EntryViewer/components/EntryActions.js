import React, {Fragment} from  'react';
export const EntryActions = (props) => {
	const {
		onView,
		onDownload,
		onResend
	} = props;
	return(
		<Fragment>
			<button
				onClick={onView}
			>
				View
			</button>
			<button
				onClick={onDownload}
			>
				Download
			</button>
			<button
				onClick={onResend}
			>
				Resend
			</button>

		</Fragment>
	)
}