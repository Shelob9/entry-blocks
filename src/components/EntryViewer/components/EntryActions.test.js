import renderer from "react-test-renderer";
import React from "react";
import {EntryActions} from "./EntryActions";

describe( 'EntryActions component ', () => {
	const onView =jest.fn();
	const onDownload =jest.fn();
	const onResend =jest.fn();
	test('EntryActions matches snapshot', () => {
		expect(
			renderer.create(
				<EntryActions
					onView={onView}
					onResend={onResend}
					onDownload={onDownload}
				/>
			)
		).toMatchSnapshot();
	});

});
