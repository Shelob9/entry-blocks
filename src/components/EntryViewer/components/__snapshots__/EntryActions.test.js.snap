exports[`EntryActions component  EntryActions matches snapshot 1`] = `
Array [
  <button
    onClick={[Function]}>
    View
</button>,
  <button
    onClick={[Function]}>
    Download
</button>,
  <button
    onClick={[Function]}>
    Resend
</button>,
]
`;
